faq.html

```diff
@@ -175,6 +144,111 @@ <h1>How can I use different highlighting for tokens with the same name in differ
 			inside: h1.parentNode
 		});
 	});
+
+	function printTokens(grammar) {
+		function toArray(x) {
+			return Array.isArray(x) ? x : (x == undefined ? [] : [x]);
+		}
+
+		var lines = [];
+		function log(line) {
+			if (lines.indexOf(line) === -1) {
+				lines.push(line);
+			}
+		}
+
+		var languageMap = new Map();
+		Object.keys(Prism.languages).filter(function (l) { return l in components.languages; }).forEach(function (l) {
+			languageMap.set(Prism.languages[l], 'Prism.languages["' + l + '"]');
+		});
+
+		var stack = new Map();
```

Comment:
If this project supports IE11, I think the website should too.

Reply:
I agree. This is why I only used the part of the Map API which is [supported by IE11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Browser_compatibility).

As I said:
> Everything works as intended in Chrome, Firefox, Edge, and IE11.

Reply:
Oh, I didn't realize Map was supported in IE11.